hack/common.sh

```diff
@@ -1001,6 +1001,23 @@ function os::build::find-binary() {
 }
 readonly -f os::build::find-binary
 
+# os::build::get-go-bin-location-from-gopath locates a go install binary in GOPATH directories 
+# and returns the first path where the binary was found. 
+function os::build::get-go-bin-location-from-gopath() {
```

Comment:
Since this function does not point to binaries created by builds of Origin, we should locate it in `hack/lib/util/misc.sh`